src/DssSpell.t.sol

```diff
@@ -211,10 +207,15 @@ contract DssSpellTest is DSTest, DSMath {
         SpellLike prevSpell = SpellLike(spellValues.previous_spell);
         // warp and cast previous spell so values are up-to-date to test against
         if (prevSpell != SpellLike(0) && !prevSpell.done()) {
-            hevm.warp(spellValues.previous_spell_execution_time);
-            // jump to nextCastTime to be a little more forgiving on the spell execution time
-            hevm.warp(prevSpell.nextCastTime());
-            prevSpell.cast();
+            if (prevSpell.eta() == 0) {
+                vote(address(prevSpell));
+                scheduleWaitAndCast(address(prevSpell));
+            }
+            else {
+                // jump to nextCastTime to be a little more forgiving on the spell execution time
+                hevm.warp(prevSpell.nextCastTime());
+                prevSpell.cast();
+            }
```

Comment:
@brianmcmichael I'd like to have another opinion on this before approving the review, Thanks!